martin/src/config/file/tiles/postgres/builder.rs

```diff
@@ -154,28 +153,14 @@ impl PostgresAutoDiscoveryBuilder {
                 ));
             }
 
-            let Some(db_tables) = find_info(&db_tables_info, &cfg_inf.schema, "schema", id) else {
-                continue;
-            };
-            let Some(db_geo_columns) = find_info(db_tables, &cfg_inf.table, "table", id) else {
-                continue;
-            };
-            let Some(db_inf) = find_info(
-                db_geo_columns,
-                &cfg_inf.geometry_column,
-                "geometry column",
-                id,
-            ) else {
+            let Ok(merged_inf) = self.instantiate_one_table(&db_tables_info, id, cfg_inf) else {
```

Comment:
much simpler to read, if we can get this to log the same way 👍🏻

As it is implmeneted (ignoring the error/warning/info log) this is a harder sell.